packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -78,6 +78,11 @@ class Tooltip extends React.Component {
      */
     open: PropTypes.bool,
 
+    /**
+     * Whether the popover should stay open after leaving the content
+     */
+    openOnLeave: PropTypes.bool,
```

Comment:
I would aim for something a bit more descriptive to a boolean here
`keepOpen` or something similar.

openOnLeave implies to me that the tooltip will open if I do something

Reply:
Okay how about `keepOpenOnLeave` then? too long?

Reply:
I assume the idea is that once you hover, the tooltip stays open indefinitely until closed? (I assume there's a way to close it)